Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -160,12 +160,55 @@ export function extend(publicAPI: object, model: object, initialValues?: IVolume
  */
 export function newInstance(initialValues?: IVolumeMapperInitialValues): vtkVolumeMapper;
 
+/**
+ * Helper class to perform MPR.
+ */
+
+export interface vtkSliceHelper {
```

Comment:
This needs to extends vtkObject, also have its own `newInstance` and `extend` like in VolumeMapper TS defs, maybe something like this :

```ts
export declare const vtkSliceHelper: {
	newInstance: typeof newInstanceSliceHelper,
	extend: typeof extendSliceHelper,
};
```